csharp-basics/exercises/Arrays/Arrays/FindNemo/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace FindNemo;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine(FindNemoFunc("I am Nemo and dont know anything !"));
+    }
+
+    static string FindNemoFunc(string input)
+    {
+        var words = input.Split(' ');
+        var result = "";
+        for (var index = 0; index < words.Length; index++)
+        {
+            if (words[index] == "Nemo")
+            {
+                result = "I found Nemo at " + index;
```

Comment:
Tev parādīs 0, ja būs pirmais vārds, bet prasīts -  [the order of the word you find Nemo] FindNemo("Nemo is me") ➞ "I found Nemo at 1!"